docs/source/conf.py

```diff
@@ -2796,6 +2796,7 @@
     "ConstraintViolationError",
     "DynamicDimConstraintPrinter",
     "GuardOnDataDependentSymNode",
+    "PendingUnbackedSymbolNotFound",
```

Comment:
Adding new entries in this file is NOT ok. We should document all the new public API we add!